docker/base/launchTomcat.sh

```diff
@@ -20,31 +20,38 @@ echo "!!============================================================"
-echo "        Admin username               ADMIN_USER              ${ADMIN_USER}"
-echo "        Admin password               ADMIN_PASSWORD          ${ADMIN_PASSWORD}"
-echo "        Run tomcat with debug        DEBUG                   ${DEBUG}"
-echo "        Extra firefly properties     FIREFLY_OPTS            ${FIREFLY_OPTS}"
-echo "        Shared work Area             FIREFLY_SHARED_WORK_DIR ${FIREFLY_SHARED_WORK_DIR}"
-echo "(*)  If MAX_JVM_SIZE is blank, autosizing properties INIT_RAM_PERCENT and MAX_RAM_PERCENT are used instead"
-echo "(**) Autosizing properties INIT_RAM_PERCENT and MAX_RAM_PERCENT are not used if MAX_JVM_SIZE is set"
+echo "Using Environment Variables: \n"
+echo "        Description                  Name                       Value"
+echo "        -----------                  --------                   -----"
+echo "        Admin username               ADMIN_USER                 ${ADMIN_USER}"
+echo "        Admin password               ADMIN_PASSWORD             ${ADMIN_PASSWORD}"
+echo "        Additional data path         VISUALIZE_FITS_SEARCH_PATH ${VISUALIZE_FITS_SEARCH_PATH}"
+echo
+echo "  Advanced environment variables:"
+echo "        Min JVM size (*)             MIN_JVM_SIZE               ${MIN_JVM_SIZE}${MM_NOT_USED}"
+echo "        Max JVM size (*)             MAX_JVM_SIZE               ${MAX_JVM_SIZE}${MM_NOT_USED}"
+echo "        Initial JVM size (**)        INIT_RAM_PERCENT           ${INIT_RAM_PERCENT}${PER_NOT_USED}"
+echo "        Max JVM size (**)            MAX_RAM_PERCENT            ${MAX_RAM_PERCENT}${PER_NOT_USED}"
+echo "        CPU cores (0 means guess)    JVM_CORES                  ${JVM_CORES}"
+echo "        Run tomcat with debug        DEBUG                      ${DEBUG}"
+echo "        Extra firefly properties     FIREFLY_OPTS               ${FIREFLY_OPTS}"
+echo "  (*)  If MAX_JVM_SIZE is blank, autosizing properties INIT_RAM_PERCENT and MAX_RAM_PERCENT are used instead"
+echo "  (**) Autosizing properties INIT_RAM_PERCENT and MAX_RAM_PERCENT are not used if MAX_JVM_SIZE is set"
 echo
 echo "Ports: "
 echo "        8080 - http"
 echo "        5050 - debug"
 echo
 echo "Volume Mount Points: "
-echo "        Log directory : ${CATALINA_HOME}/logs : Directory for logs files"
-echo "        Local Images  : /local/data : Root directory for images and tables that firefly can read"
+echo "    /firefly/config           : used to override application properties"
+echo "    /firefly/workarea         : work area for temporary files"
+echo "    /firefly/shared-workarea  : work area for files that are shared between multiple instances of the application"
+echo "    /firefly/logs             : logs directory"
+echo "    /firefly/logs/statistics  : directory for statistics logs"
+echo "    /firefly/alerts           : alerts monitor will watch this directory for application alerts"
+echo "    /external                 : default external data directory visible to Firefly"
```

Comment:
maybe we want to break these into two parts as well, common and advanced.
`logs`, `shared-workarea`, `/external` as common and the others as advanced.